src/pint/models/expdip.py

```diff
@@ -0,0 +1,286 @@
+                name=f"EXPDIPIDX_{index}",
+                units="",
+                value=gamma,
+                description="Chromatic exponential dip index",
+                parameter_type="float",
+                frozen=frozen,
+                tcb2tdb_scale_factor=1,
+                prefix_aliases=["EXPINDEX_"],
+            )
+        )
+
+        if isinstance(tau, u.Quantity):
+            tau = tau.to_value(u.s)
+
+        self.add_param(
+            prefixParameter(
+                name=f"EXPDIPTAU_{index}",
+                units="day",
+                value=tau,
+                description="Chromatic exponential dip decay timescale",
+                parameter_type="float",
+                frozen=frozen,
+                tcb2tdb_scale_factor=1,
+                prefix_aliases=["EXPTAU_"],
+            )
+        )
+
+        self.setup()
+        self.validate()
+
+        return index
+
+    def remove_exp_dip(self, index: Union[float, int, List[int], np.ndarray]) -> None:
+        """Removes all exp dip parameters associated with a given index/list of indices.
+
+        Parameters
+        ----------
+
+        index : float, int, list, np.ndarray
+            Number or list/array of numbers corresponding to DMX indices to be removed from model.
+        """
+
+        if isinstance(index, (int, float, np.int64)):
+            indices = [index]
+        elif isinstance(index, (list, set, np.ndarray)):
+            indices = index
+        else:
+            raise TypeError(
+                f"index must be a float, int, set, list, or array - not {type(index)}"
+            )
+        for index in indices:
+            index_rf = f"{int(index):d}"
+            for prefix in ["EXPDIPEP_", "EXPDIPAMP_", "EXPDIPTAU_", "EXPDIPIDX_"]:
+                self.remove_param(f"{prefix}{index_rf}")
+        self.validate()
+
+    def get_indices(self) -> np.ndarray:
+        """Returns an array of integers corresponding to exp dip parameters.
+
+        Returns
+        -------
+        inds : np.ndarray
+            Array of exp dip indices in model.
+        """
+        inds = [int(p.split("_")[-1]) for p in self.params if "EXPDIPEP_" in p]
+        return np.array(inds)
+
+    def setup(self) -> None:
+        super().setup()
+        # Get DMX mapping.
+        # Register the DMX derivatives
+        for prefix_par in self.get_params_of_type("prefixParameter"):
+            if prefix_par.startswith("EXPDIPEP_"):
+                self.register_deriv_funcs(self.d_delay_d_T, prefix_par)
+            elif prefix_par.startswith("EXPDIPAMP_"):
+                self.register_deriv_funcs(self.d_delay_d_A, prefix_par)
+            elif prefix_par.startswith("EXPDIPTAU_"):
+                self.register_deriv_funcs(self.d_delay_d_tau, prefix_par)
+            elif prefix_par.startswith("EXPDIPIDX_"):
+                self.register_deriv_funcs(self.d_delay_d_gamma, prefix_par)
+
+    def get_ffac(self, toas: TOAs) -> np.ndarray:
```

Comment:
Is there ever a time when you might want to change this?  Maybe for LOFAR data or something where 1400 MHz is far outside the band?